gym/utils/env_checker.py

```diff
@@ -281,7 +234,6 @@ def check_env(env: gym.Env, warn: bool = None, skip_render_check: bool = False):
     # ==== Check the reset method ====
     check_reset_seed(env)
     check_reset_options(env)
-    check_reset_info(env)
```

Comment:
Should we replace this with a signature check?

Reply:
added, but I also brought back check_reset_info. Open to feedback if the current tests seem redundant since there is some redundancy between test_reset_info and test_passive_env_reset_checker